src/openforms/submissions/tests/form_logic/test_submission_logic.py

```diff
@@ -369,6 +372,114 @@ def test_check_logic_hide_with_default_value(self):
             data["step"]["formStep"]["configuration"]["components"][0]["hidden"],
         )
 
+    def test_check_logic_with_registration_backend_correctly_displayed_in_admin(self):
```

Comment:
can you please put the tests in `src/openforms/submissions/tests/test_admin.py` with the rest of the admin tests, and use webtest?

These tests here are tests for API endpoints

Reply:
Hmm yeah i wasn't sure about the placement of this test.

On the one hand, it is about the admin, so should be placed with the admin tests. On the other hand, this test heavily focuses on the form logic, and the effects it has..

Reply:
But I'll move it to the `test_admin` 👍

Reply:
> On the one hand, it is about the admin, so should be placed with the admin tests. On the other hand, this test heavily focuses on the form logic, and the effects it has..

It's testing what is displayed on the admin page and that's what the initial comment was about :-) that the logic engine needs to be used is an implementation detail, but the original test case was `APITestCase` and that's _definitely_ the wrong location for admin tests :stuck_out_tongue: